### PR TITLE
Add get-all-relevant-apartments method

### LINKED ISSUE #122: Add get all relevant apartments method
- Add a class method to fetch all apartment details from the DB by the `relevant` field.
- Add a test to validate this class method.

### PR DESCRIPTION
**Pull request by tamirho**
- Add a class method to fetch all apartment details from the DB by the relevant field.
- Add a test to validate this class method.

Fix #122

### CODE REVIEW

**Review by simzacks — APPROVED** [review]

File: apartments/tests.py [hunk 1]
```diff
@@ -61,6 +61,11 @@ def test_get_apartment(self, new_apartment):
         check_apartment = Apartment.objects.get(owner=new_apartment.owner)
         assert check_apartment == new_apartment
 
+    def test_get_all_relevant_apartments(self):
+        apartments_qurey = Apartment.get_all_relevant_apartments()
+        assert all(isinstance(current_apartment, Apartment) for current_apartment in apartments_qurey)
+        assert all(current_apartment.is_relevant is True for current_apartment in apartments_qurey)
```
Comment (Omeramsc) [inline apartments/tests.py]: Pretty sure `is True` can be discarded here (Truthy value).

File: apartments/tests.py [hunk 2]
```diff
@@ -61,6 +61,11 @@ def test_get_apartment(self, new_apartment):
         check_apartment = Apartment.objects.get(owner=new_apartment.owner)
         assert check_apartment == new_apartment
 
+    def test_get_all_relevant_apartments(self):
+        apartments_qurey = Apartment.get_all_relevant_apartments()
+        assert all(isinstance(current_apartment, Apartment) for current_apartment in apartments_qurey)
```
Comment (Omeramsc) [inline apartments/tests.py:74]: I don't see any insertion of test data into the database.
what would happen if there no apartments in it?

**Review by Omeramsc — CHANGES_REQUESTED** [review]

File: apartments/tests.py (hunk 1 above)
Comment (tamirho) [reply apartments/tests.py]: Got it

File: apartments/tests.py (hunk 2 above)
Comment (tamirho) [reply apartments/tests.py:74]: I thought the test data was used for that.. but I understand from you that this is not happening.
So I will add insertion of test data.

Comment (Omeramsc) [reply apartments/tests.py:74]: Please double-check me on that.

Comment (simzacks) [reply apartments/tests.py:74]: Generally, we accept that the test data inserted into the system is expected to be there from the tests perspective.

Comment (Omeramsc) [reply apartments/tests.py:74]: ack, I'm resolving this, but fix the conflicts so we'll be able to merge @tamirho

**Comment by AmitAharoni** [comment]
LGTM

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**